clearwater-infrastructure/usr/share/clearwater/bin/set_ntp_ip

```diff
@@ -0,0 +1,67 @@
+#Copyright (C) 2016 Metaswitch Networks Ltd
+
+import os, pipes, platform, subprocess, sys
+from datetime import datetime
+
+NTP_CONFIG="/etc/ntp.conf"
+KEY="server"
+SCRIPT="/usr/share/clearwater/bin/set_ntp_ip"
+
+def set_ntp_ip(ntp_ip_address):
+
```

Comment:
I don't think we'd normally have a blank line at the start of a function (c.f. https://github.com/Metaswitch/clearwater-etcd/blob/dev/src/metaswitch/clearwater/cluster_manager/etcd_synchronizer.py#L54)